pkg/sql/plan/function/builtin/unary/math.go

```diff
@@ -19,12 +19,13 @@ import (
 	"github.com/matrixorigin/matrixone/pkg/container/types"
 	"github.com/matrixorigin/matrixone/pkg/container/vector"
 	"github.com/matrixorigin/matrixone/pkg/encoding"
-	"github.com/matrixorigin/matrixone/pkg/vectorize/acos"
+	"github.com/matrixorigin/matrixone/pkg/vectorize/momath"
 	"github.com/matrixorigin/matrixone/pkg/vm/process"
-	"golang.org/x/exp/constraints"
 )
 
-func Acos[T constraints.Integer | constraints.Float](vs []*vector.Vector, proc *process.Process) (*vector.Vector, error) {
+type mathFn func(*vector.Vector, *vector.Vector) error
+
+func math1(vs []*vector.Vector, proc *process.Process, fn mathFn) (*vector.Vector, error) {
```

Comment:
it's not a good function name.

mathCommonFn?

Reply:
It is mathfunction with 1 arg.    I may have math2 soon.